### DISCUSSION TITLE
Kamstrup DK-8660, Dala energi, Sweden

### DISCUSSION THREAD

**Discussion by Wergilius** [UtilitechAS/amsreader-firmware#446] [Show and tell]
Confirmation Pow-k+ work fine with Kamstrup DK-8660 with operator Dala Energi, Sweden
![image](https://user-images.githubusercontent.com/43338576/218458233-e7184746-9fdf-4f6b-a893-d19691ce8f37.png)

Communication 2400, 8N1, no encryption

**Comment by ArnieO** [comment]
As expected - but thank you for confirming! 👍

**Comment by Wergilius** [comment]
However, I got this error but not sure if and where the "error" is located:

Accumulated reactive export, sensor.d010_tqo
![image](https://user-images.githubusercontent.com/43338576/218528670-9cc887bc-a607-4a27-9494-692c72c1e8a1.png)

Looking in the description of the protocol, the meassurement is correct:
![image](https://user-images.githubusercontent.com/43338576/218529081-97dce474-e25c-4e65-bda2-d3299651a77c.png)

Assume both items would have the same error, however one is deactivated.

**Reply by ArnieO** [reply]
There are several users having issues with the Home Assistant integration following an update of Home Assistant. Please search for Home Assistant under "Issues" and read relevant threads there.

**Reply by gskjold** [reply]
Judging from HA documentation (https://developers.home-assistant.io/docs/core/entity/sensor/#available-device-classes) it doesn't actually support kvarh (just as the error says), so my implementation is indeed not valid for accumulated reactive energy. Does that actually break the sensor? If so, I may have to remove the unit
